setup/README.md

```diff
@@ -0,0 +1,88 @@
+# Orka VM Setup Script Usage
+
+## This script automates the configuration and setup of macOS Orka VMs for optimal performance. It configures system settings, installs required tools, and prepares the VM for use with Orka
+
+### System requirements
+
+- macOS IPSW file (Tahoe, Sequoia, or Sonoma)
+- Admin/sudo privileges on the VM
+- Internet connection for downloading packages
+- Terminal access (via SSH, Screen Sharing, or direct console)
+
+### Before running
+
+- Ensure you have administrator access to the VM
+- Close any important applications (they will be terminated during cleanup)
+- Save any work in progress (the VM will reboot automatically)
+
+### Installation
+
+#### Download via GitHub
+
+```bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/macstadium/orka-images/setup/setup.sh)"
```

Comment:
```suggestion
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/macstadium/orka-images/HEAD/setup/setup.sh)"
```

I did not catch this before - but after we merge the PR, this should be the correct path